spec/support/flakyTests.json

```diff
@@ -0,0 +1,4 @@
+[
+  "flaky example",
```

Comment:
Do we need this line?

Reply:
It's for this test, how else would we know if this retry logic works.

https://github.com/parse-community/parse-server/pull/9218/files#diff-ea844417e94f3b896164374bb3ab8b4756cacb50dae1e6832b6966af4f3302f0

Reply:
We shouldn't mix meta data with test data.  Could you write the required file content as part of the test?